src/PrestaShopBundle/Resources/views/Admin/Improve/International/Tax/Blocks/form.html.twig

```diff
@@ -35,31 +36,8 @@
     <div class="card-text">
       {{ form_errors(taxForm) }}
 
-      {% set nameHint %}
-        {{ 'Tax name to display in carts and on invoices (e.g. "VAT").'|trans({}, 'Admin.International.Help') }}
-        {{ 'Invalid characters:'|trans({}, 'Admin.Notifications.Info') ~ ' <>;=#{}' }}
-      {% endset %}
-
-      {{ ps.form_group_row(taxForm.name, {}, {
-        'label': 'Name'|trans({}, 'Admin.Global'),
-        'help': nameHint,
-      }) }}
-
-      {% set rateHint %}
-        {{ 'Format: XX.XX or XX.XXX (e.g. 19.60 or 13.925)'|trans({}, 'Admin.International.Help') }}
-      {% endset %}
-
-      {{ ps.form_group_row(taxForm.rate, {}, {
-        'label': 'Rate'|trans({}, 'Admin.International.Feature'),
-        'help': rateHint,
-      }) }}
-
-      {{ ps.form_group_row(taxForm.is_enabled, {}, {
-        'label': 'Enable'|trans({}, 'Admin.Actions')
-      }) }}
-
       {% block tax_form_rest %}
-        {{ form_rest(taxForm) }}
+        {{ form_widget(taxForm) }}
```

Comment:
Maybe the block name should be `tax_form_widget` ? Based on what you did for `attachment_form_widget`